server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -1020,10 +1020,13 @@ public void testStartSearchNode() throws Exception {
         // test start node without search role
         internalCluster().startNode(Settings.builder().put(onlyRole(DiscoveryNodeRole.DATA_ROLE)));
         // test start non-dedicated search node with TIERED_REMOTE_INDEX feature enabled
-        internalCluster().startNode(
-            Settings.builder()
-                .put(onlyRoles(Set.of(DiscoveryNodeRole.SEARCH_ROLE, DiscoveryNodeRole.DATA_ROLE)))
-                .put(TIERED_REMOTE_INDEX, true)
+        assertThrows(
+            SettingsException.class,
```

Comment:
Why does this throw an exception instead of use the default value for `node.search.cache.size`? Maybe could use a comment in the test explaining the same.

Reply:
there is already a comment explaining that here: https://github.com/opensearch-project/OpenSearch/blob/a785073e5e7925ef8e5605427cae943822100f4a/server/src/main/java/org/opensearch/node/Node.java#L2036

but thought more about it and putting a feature flag here doesnt make sense, would remove it